themes/glowsticks.omp.yaml

```diff
@@ -52,7 +52,7 @@ blocks:
           always_enabled: true
           style: round
         style: powerline
-        template: " 羽{{ .FormattedMs }} "
+        template: " 󰔟 {{ .FormattedMs }} "
```

Comment:
we need the UTF notation of the icons instead the actual icon: `\uXXXX`

Reply:
why?

Reply:
This isn't readable. When you export the config again that "should" be done for you btw.

Reply:
I used `omp config export` and this happened:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/70787919/ca2e7bb6-d795-419f-977f-1d3407b0a150)